modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -54,6 +54,11 @@
 	<td>//div[@class='autofit-section' and contains(.,'${key_translationOption}')]//..//..//span[@class="label label-${key_translationStatus}"]</td>
 	<td></td>
 </tr>
```

Comment:
You could have created a path to the modals, why? With that you can be sure that the correct modal is openned, not just any modal.